tests/test_tutorial/test_testing/test_main_b.py

```diff
@@ -0,0 +1,10 @@
+from docs_src.app_testing import test_main_b
+
+
+def test_app():
+    test_main_b.test_create_existing_item()
+    test_main_b.test_create_item()
+    test_main_b.test_create_item_bad_token()
+    test_main_b.test_read_inexistent_item()
+    test_main_b.test_read_item()
+    test_main_b.test_read_item_bad_token()
```

Comment:
This doesn't duplicate the tests as well?

Reply:
If I remember correctly, the tests from `docs_src` are not run by pytest, so the way to run them and have coverage for that is to do this.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.

Reply:
I remember doing this because before my changes here, the tests were running twice, and I think (I didn't check) this makes them run twice, because (I think) the import is enough to make pytest find the tests. Anyway... :eyes:

Reply:
Hum, maybe I have to review this again.

Although, one possible explanation is if you were running `pytest` and I was running `pytest tests`.

But anyway, yeah, maybe for later. 😅